examples/fargate/install-metrics.sh

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env bash
+
+export CLUSTER=sumologic-demo
+export PROFILE_NAME=sumologic
+export NAMESPACE=sumologic
+export AWS_REGION=us-east-2
+export HELM_INSTALLATION_NAME=collection
+export SG_NAME=sumologic-collection
+export METRIC_PODS=10
+
+## Set up Fargate Profile for Sumo Logic namespace
+eksctl create fargateprofile \
+  --cluster "${CLUSTER}" \
+  --name "${PROFILE_NAME}" \
+  --namespace "${NAMESPACE}"
+
+## Create EFS Volume
+export EFS_ID="$(
+  aws efs describe-file-systems \
+    --region="${AWS_REGION}" | \
+    jq '.FileSystems[] | 
+      select(.Tags[].Key == "Name" and .Tags[].Value == "SumologicCollectionVolumes") |
+      .FileSystemId' \
+      --raw-output)"
+
+if [[ -z "${EFS_ID}" ]]; then
+  aws efs create-file-system \
+    --encrypted \
+    --performance-mode generalPurpose \
+    --throughput-mode bursting \
+    --tags Key=Name,Value=SumologicCollectionVolumes \
+    --region "${AWS_REGION}"
+  export EFS_ID="$(
+    aws efs describe-file-systems \
+      --region="${AWS_REGION}" | \
+      jq '.FileSystems[] | 
+        select(.Tags[].Key == "Name" and .Tags[].Value == "SumologicCollectionVolumes") | 
+        .FileSystemId' \
+        --raw-output)"
+fi
+
+## Create EFS access points for metric Pods
+for (( counter=0; counter<METRIC_PODS; counter++ )); do
```

Comment:
should we have similar examples for logs and events?

Reply:
We should add the same, but for traces and events. We are not going to create PVC for logs as we don't want to install it on cluster

Reply:
Also I'm not sure if we want to create averything in one script as for now it is just dumb set of commands, so customer doesn't have control over it (what to enable, what to disable and so on)

Reply:
But to sum up. IMHO script is just addition to the documentation, and we should focus on simple documentation rather than on the script itself

Reply:
It will need it, but the docs is for metrics as for now. I would prefer to merge it and add/change in next PRs